encord/user_client.py

```diff
@@ -742,6 +747,28 @@ def deidentify_dicom_files(
             },
         )
 
+    def create_folder(
```

Comment:
I wonder if this name is clear enough. UserClient provides a number of method that are related to many different entities, so `folder` might be a bit too generic. Perhaps `create_storage_folder` is slightly more clear?

Reply:
I went a bit back and forth on this with myself, too. The consequence here would be that I guess we'd have to call the entity classes `StorageFolder` and `StorageItem` then? And we don't have any other kind of `Folder`.

Thinking of this again, feels a bit verbose, but probably a good idea for `Item`, as that one *is* too generic.

Let me think for 5 more minutes ;)

Reply:
Actually yes ;), let me rename all that stuff.